spec/output_spec.rb

```diff
@@ -25,6 +26,7 @@
       admin/config.yml
       assets/css/main.css.map
       assets/js/main.js.LICENSE.txt
+      assets/js/contact.js.LICENSE.txt
```

Comment:
Am I correct that we used the contact form temporarily for demonstration, and it won't live there long-term? If so, should we remove this allowlist entry if it'll never exist in the final implementation?

Reply:
@sheldon-b It looks like that commit might have been made to a different branch than this pull request.